Validation/RecoEgamma/python/egammaValidationMiniAOD_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms

```

Comment:
sounds good to me, but then I would ask you to incorporate in this sequence the photon miniAOD validation as well so that one sequence only is added to the top prevalidationMiniAOD